packages/emails/email-manager.ts

```diff
@@ -81,6 +81,38 @@ export const sendRescheduledEmails = async (calEvent: CalendarEvent) => {
   await Promise.all(emailsToSend);
 };
 
+export const sendScheduledSeatsEmails = async (
+  calEvent: CalendarEvent,
+  invitee: Attendee,
+  newSeat: boolean
+) => {
+  const emailsToSend: Promise<unknown>[] = [];
+
+  emailsToSend.push(
+    new Promise((resolve, reject) => {
+      try {
+        const scheduledEmail = new AttendeeScheduledEmail(calEvent, invitee);
+        resolve(scheduledEmail.sendEmail());
+      } catch (e) {
+        reject(console.error("AttendeeRescheduledEmail.sendEmail failed", e));
+      }
+    })
+  );
```

Comment:
Change is rather than going through all attendees, just send an email to the new attendee so previous ones don't get spammed.